src/GUI/Shared/Header.razor.css

```diff
@@ -0,0 +1,51 @@
+﻿.header {
+    width: 100%;
+    height: 75px;
+
+    background-color: #7AC7FF;
+    display: flex;
+    align-items: center;
+}
+
+.header input {
```

Comment:
I think it will be better if you use bootstrap inputs https://getbootstrap.com/docs/4.0/components/input-group/
for example <input type="text" class="form-control"> then no need create styles for input

Reply:
Fixed